serpentTools/parsers/sensitivity.py

```diff
@@ -389,7 +393,7 @@ def plot(self, resp, zai=None, pert=None, mat=None, sigma=3,
 
         errors = resMat[..., 1] * values * sigma
 
-        energies = self.energies * 1E6
+        energies = self.energies * 1E6 if mevscale is False else self.energies
```

Comment:
Identical but a little easier to follow
```suggestion
        energies = self.energies if mevscale else self.energies * 1E6
```